src/main/resources/isoAppHeadValidator/iso-apphead-validator.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<template xmlns="http://ws.apache.org/ns/synapse" name="isoAppHeadValidator">
+    <sequence>
+        <property expression="boolean($body//BizMsgEnvlp/app:AppHdr)" name="uri.var.IsAppHdrExists" action="set" scope="default" type="BOOLEAN" xmlns:app="urn:iso:std:iso:20022:tech:xsd:head.001.001.03"/>
+        <switch source="$ctx:uri.var.IsAppHdrExists">
+            <case regex="true">
+                <log level="custom">
```

Comment:
Do we need this log?

Reply:
Removed in 50525032ea8f18daabcae7e87bc834e2202297c1